implementations/utils/tpu.py

```diff
@@ -223,7 +223,7 @@ def wait_for_queued_resource_deletion(op_name: Optional[str]):
 
 
 @task
-def ssh_tpu(
+def ssh_host(
```

Comment:
nit: keep the name `ssh_tpu` or similar. The semantics are different than `gpu.ssh_host`, since this will SSH to _multiple_ hosts in a TPU resource.